src/pytorch_lightning/trainer/trainer.py

```diff
@@ -654,12 +654,16 @@ def _call_and_handle_interrupt(self, trainer_fn: Callable, *args: Any, **kwargs:
             if not self.interrupted:
                 self.state.status = TrainerStatus.INTERRUPTED
                 self._call_callback_hooks("on_exception", exception)
+                for logger in self.loggers:
+                    logger.finalize("failed")
```

Comment:
If this gets moved here, then it shoud be removed from `Trainer._call_teardown_hook()`, right?